examples/asset/extra_source.rs

```diff
@@ -0,0 +1,47 @@
+//! Example of loading assets from an extra asset source
```

Comment:
I don't think the term "extra source" is immediately clear. Can you expand on it here?

Reply:
The problem with `custom` would be that this asset source isn't custom, it's just the default asset source targeting another folder. Of course the same code could be used as a basis for a fully custom asset source